src/install/demo/dropdown1/default.mdx

```diff
@@ -0,0 +1,65 @@
+---
+componentType: default
+headingText: Choose your download option
+---
+
+You can use the tabs component to show a set of horizontal choices.
+
+<Tabs>
+  <TabsBar>
+    <TabsBarItem id="tab">Tab example</TabsBarItem>
+    <TabsBarItem id="powershell">Powershell</TabsBarItem>
+    <TabsBarItem id="web">Web</TabsBarItem>
+  </TabsBar>
+
+  <TabsPages>
+    <TabsPageItem id="tabs">
+        ```react
+        <Tabs>
+            <TabsBar>
+                <TabsBarItem id="curl">Tab example</TabsBarItem>
+                <TabsBarItem id="powershell">Powershell</TabsBarItem>
+                <TabsBarItem id="web">Web</TabsBarItem>
+            </TabsBar>
+
+            <TabsPages>
+                <TabsPageItem id="curl">
+                Download the Java APM agent with this `curl` command and then unzip it.
+
+                ```sh
+                curl -0 https://download.newrelic.com/newrelic/java-agent/newrelic-agent/current/newrelic-java.zip
```

Comment:
@paperclypse visually this doesn't make sense in the build. It shows up as a link below the tab example experience. Could you provide a little more context for this?

Reply:
Yeah, I was trying to create a codeblock that showed the code for creating tabs, but I got lost in my own circular reasoning/code here :)